bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/productcarousel/ProductCarouselImpl.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2019 Adobe.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.adobe.cq.commerce.core.components.internal.models.v1.productcarousel;
+
+
+import com.adobe.cq.commerce.core.components.internal.models.v1.MagentoGraphqlClient;
+import com.adobe.cq.commerce.core.components.internal.models.v1.Utils;
+import com.adobe.cq.commerce.core.components.internal.models.v1.productlist.ProductListItemImpl;
+import com.adobe.cq.commerce.core.components.models.productcarousel.ProductCarousel;
+import com.adobe.cq.commerce.core.components.models.productlist.ProductListItem;
+import java.util.Arrays;
```

Comment:
Apply some import formatting, they should be sorted, ever IDE does that.

Reply:
Net Beans IDE formating used